test/dlc_tests/container_tests/bin/testOpenCV.py

```diff
@@ -1,20 +1,28 @@
 import cv2
```

Comment:
Is the change compatible with previous framework version DLC?

Reply:
I don't see why it shouldn't

Reply:
@hsl89 it looks like this script takes in an argument now. It's sort of unclear if one came back to this that the arg is supposed to be an image file.

Can you use something like "argparse" instead of an unnamed sys arg?

Also, re: tejas' comment - can you provide more information about what these changes actually are, and why they're compatible with prior framework versions